benchmarks/BUILD

```diff
@@ -1,20 +1,33 @@
 licenses(["notice"])  # Apache 2
 
-load(
-    "@envoy//bazel:envoy_build_system.bzl",
-    "envoy_package",
-)
+load("@rules_python//python:defs.bzl", "py_binary", "py_test")
 
-envoy_package()
+py_binary(
+    name = "benchmarks",
+    srcs = [
+        "benchmarks.py",
+        "infra.py",
+    ],
+    data = [
+        "configurations/envoy_proxy.yaml",
+    ],
+    deps = [
+        "//test/integration:integration_test_base_lean",
+    ],
+)
 
 py_test(
-    name = "benchmarks",
+    name = "test_benchmarks",
     srcs = [
         "benchmarks.py",
-        "test_benchmark.py",
+        "infra.py",
+        "test/test_discovery.py",
+    ],
+    data = [
+        "configurations/envoy_proxy.yaml",
```

Comment:
Is it possible to depend on `:benchmarks` directly here to avoid having to redeclare things?

Reply:
I improved this, but could not eliminate all of the duplication, ptal